src/sentry/templates/sentry/js-sdk-loader.ts

```diff
@@ -60,6 +62,22 @@ declare const __LOADER__IS_LAZY__: any;
     });
   }
 
+  // We detect ES6 support by checking if module scripts are supported
+  let _hasEs6Support: boolean | undefined;
+  function hasEs6Support(): boolean {
```

Comment:
m: could we check if `Symbol` is defined as an es5 check? 

also maybe https://gist.github.com/bendc/d7f3dbc83d0f65ca0433caf90378cd95 works better.

Reply:
Hmm, not sure if `new Function("(a = 0) => a");` works well with all CSP security settings etc...? isn't this basically eval, which it may complain about?

but I guess we can do `'Symbol' in window` check, that should be fine I guess? 

FWIW I think [module](https://caniuse.com/es6-module) support should be a pretty good proxy. [Symbol](https://caniuse.com/mdn-javascript_builtins_symbol) is a bit "better" supported.

Reply:
injecting an inline script seems more likely to trigger CSP fwiw

Reply:
good point, we should def. not do that then 😬 so let's go the symbol check route then, wdyt? should this work:

```js
try {
  Symbol("loader");
} catch(error) {
  // this is ES5
}
```

Reply:
This might fail with polyfills but maybe that is ok because it represents that someone is already polyfilling the necessary es6 features.

I'm comfortable with shipping this.

Reply:
Yeah, I think if `Symbol` is polyfilled we can ~assume that ES6 stuff is polyfilled generally. Overall by now ES6 is so widely supported that I figure only very few will be affected by this overall!